src/App.tsx

```diff
@@ -1,22 +1,51 @@
 import { useMemo } from 'react'
-import { BrowserRouter, Routes, Route } from 'react-router-dom'
+import { createBrowserRouter, RouterProvider } from 'react-router-dom'
 import Home from './components/templates/Home'
 import Contact from './components/templates/Contact'
 import { VideoListPage } from './components/templates/VideoListPage'
 import SeriesPage from './components/templates/SeriesPage'
 import AnimationPage from './components/templates/AnimationPage'
-import { Header } from './components/molecules/headers/Header'
 import { ThemeProvider } from '@mui/material/styles'
 import { kurtTheme } from './theme'
 import { Footer } from './components/organisms/Footer/Footer'
+import { KurtApp } from './components/pages/kurtApp/KurtApp'
+
+const router = createBrowserRouter([
```

Comment:
Create the route tree outside of the app context

Reply:
This is very cool!
I just wrote it the way I've used it in the past, but I think I might like this way better.

Reply:
Yup, I'd never seen this before. I was trying to figure out how to pass args from the Router to a Component and saw this page saying "use this unless you can't"